test/intl402/Temporal/PlainDate/from/chinese-calendar-dates.js

```diff
@@ -0,0 +1,94 @@
+// Copyright 2025 Google Inc, Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindate.from
+description: Test `from` with Chinese calendar
+features: [Temporal]
+---*/
+
+const testChineseData = new Date("2001-02-01T00:00Z").toLocaleString("en-US-u-ca-chinese", {
+  day: "numeric",
+  month: "numeric",
+  year: "numeric",
+  era: "short",
+  timeZone: "UTC"
+});
+const hasOutdatedChineseIcuData = !testChineseData.endsWith("2001");
+
+const calendar = "chinese";
+
+const chineseYearOffset = new Temporal.PlainDate(1, 1, 1, calendar).year;
```

Comment:
Same as above, we should not permit alternate arithmetic years for the Chinese calendar. I think in most (all?) of these cases we can just assume `chineseYearOffset` is 0.

Reply:
Fixed in [3f8f514](https://github.com/tc39/test262/pull/4667/commits/3f8f5149f22ae8bd5572ba26bf693d594da89af9)